### ISSUE TITLE
Make command

### ISSUE THREAD

**Issue by reno777** [reno777/nenya#3]
Try and make it a useable command, ei: `nenya` with shell before moving to python

**Comment by picatz** [comment]
mfw before moving to python 😂

**Comment by reno777** [comment]
Yeah man lol, Id like to see how to make a bash script a usable command in linux before moving to python which is ultimately better for making command line commands. I gotta see how to let users have flags and stuff too.
